### PR TITLE
[ADD] Added account_credit_control_payment_details module

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: account_credit_control_payment_details/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+=============
+
+There is nothing to configure.
+
+Known issues / Roadmap
+======================
+
+ * No known issues.
+
+Bug Tracker
```
Comment (tafaRU) [inline account_credit_control_payment_details/README.rst:41]: Please use updated version, see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker as reference

File: account_credit_control_payment_details/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+This module depends on:
+
+* account_credit_control
+
+Configuration
+=============
+
+There is nothing to configure.
+
```
Comment (tafaRU) [inline account_credit_control_payment_details/README.rst:22]: Please add https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage

File: account_credit_control_payment_details/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline account_credit_control_payment_details/__init__.py]: Please replace © by Copyright, see https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L2 as reference.

File: account_credit_control_payment_details/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd
```
Comment (tafaRU) [inline account_credit_control_payment_details/__openerp__.py]: Same as above

File: account_renumber/test/create_moves.py [hunk 5]
```diff
@@ -90,10 +90,12 @@ def create_lots_of_account_moves(dbname, user, passwd, howmany):
                               u'account.move', 'button_validate',
                               [move_id], {})
 
+
```
Comment (tafaRU) [inline account_renumber/test/create_moves.py]: Better open another PR, please remove it.

File: account_renumber/test/create_moves.py [hunk 6]
```diff
@@ -90,10 +90,12 @@ def create_lots_of_account_moves(dbname, user, passwd, howmany):
...
 # ------------------------------------------------------------------------
 # ------------------------------------------------------------------------
 # ------------------------------------------------------------------------
 
+
```
Comment (tafaRU) [inline account_renumber/test/create_moves.py]: Same as above.

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: account_credit_control_payment_details/README.rst (hunk 2 above)
Comment (JayVora-SerpentCS) [reply account_credit_control_payment_details/README.rst:22]: done

File: account_credit_control_payment_details/README.rst (hunk 1 above)
Comment (JayVora-SerpentCS) [reply account_credit_control_payment_details/README.rst:41]: done

File: account_credit_control_payment_details/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account Credit Control Payment Details",
+    "summary": "Account Credit Control Payment Details",
+    "version": "8.0.1.0.0",
+    "category": "Accounting",
+    "description": "Account Credit Control Payment Details",
```
Comment (tafaRU) [inline account_credit_control_payment_details/__openerp__.py]: Please remove it so that README file will be loaded. Thanks.

**Comment by tafaRU** [comment]
Please also add unitest per module, see https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#tests as reference.

**Comment by tafaRU** [comment]
@JayVora-SerpentCS, please check the following lint errors: https://travis-ci.org/OCA/account-financial-tools/jobs/187135154#L326-L327

Thank you in advance.

**Pull request closed without merging**